shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -114,10 +118,18 @@ FlutterWindowsEngine::FlutterWindowsEngine(const FlutterProjectBundle& project)
   plugin_registrar_ = std::make_unique<FlutterDesktopPluginRegistrar>();
   plugin_registrar_->engine = this;
 
+  messenger_wrapper_ = std::make_unique<BinaryMessengerImpl>(messenger_.get());
   message_dispatcher_ =
       std::make_unique<IncomingMessageDispatcher>(messenger_.get());
   window_proc_delegate_manager_ =
       std::make_unique<Win32WindowProcDelegateManager>();
+
+  // Set up internal channels. In the longer term, these should be replaced with
+  // formal embedder APIs.
```

Comment:
Not sure if there's an existing bug for this, but if so, consider adding a link to it here and marking this as TODO. If not, we should probably file one; or maybe one for each channel.

Reply:
Done